climate-data.md

```diff
@@ -0,0 +1,37 @@
+
+# Climate open data
+
+Everyone can have access to climate data, however many people were not able to access the data due to various reasons: failing register on the website, 
+formulating and sending request to get the right data they need.
+Here we write precise 
```

Comment:
I'm not sure if this sentence fits in.

Reply:
indeed i suggest to remove it and just add that "Detailed guide for searching and downloading climate data is sometimes lacking. "